typescript/web/src/components/layout/top-bar/user-menu.tsx

```diff
@@ -234,9 +234,9 @@ const UserMenuList = (props: UserProps) => {
           <MenuDivider />
         </>
       )}
-      <UserMenuGroup />
+      <PreferencesMenuGroup /> {/*changed*/}
       <MenuDivider />
-      <PreferencesMenuGroup />
+      <UserMenuGroup /> {/*changed*/}
```

Comment:
```suggestion
      <UserMenuGroup />
```

Reply:
Sorry for those comments. I have pushed the code after removing the comments. Please check it.